GraphApp/frmMain.cs

```diff
@@ -144,11 +138,10 @@ private static async Task<User> GetMeAsync()
 
         private static async Task<Stream> GetMePhotoAsync()
         {
-            Stream profilePhotoStream = null;
             try
             {
                 graphClient = GraphServiceClientProvider.GetAuthenticatedClient();
-                profilePhotoStream = await graphClient.Me.Photo.Content.Request().GetAsync();
+                var profilePhotoStream = await graphClient.Me.Photo.Content.Request().GetAsync();
```

Comment:
Use types as opposed to inference, again makes the code readable.

Reply:
Readability is subjective, I actually find using 'var' much more readable than using explicit typing.

Reply:
Avoid var when the type is not apparent from the right side of the assignment. You could checkout [C# Coding Conventions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions)